### ISSUE TITLE
Strange warnings from Shell script 2

### ISSUE THREAD

**Issue by jamespblloyd** [qqwang-berkeley/JUM#7]
Hey,

So I got a lot of strange warnings from running the second shell script. I am not sure what part is generating the warnings and how big a problem they are. But I do know that at the end of shell script 3 I get a LOT of errors and the final output files are nearly all empty. I am not sure if that problem is related to this or not but thought I would check what is is about (some of the whitespace has been removed from the warnings to make it easier to read). 

bash /lab/app/JUM/JUM_170323/JUM_1.3.1/JUM_2.sh /lab/app/JUM/JUM_170323/JUM_1.3.1 5 2 150
Preparing for non-intron retention AS events analyses...
6 input samples for JUM analyses
Preparing for intron retention AS events analyses...
Processing junction overhangs for Sample S100... Success.
Processing junction overhangs for Sample S104... Success.
Processing junction overhangs for Sample S108... Success.
Processing junction overhangs for Sample S102... Success.
********************
*****WARNING: -wb is ignored with -abam***************WARNING: -wb is ignored with -abam
**********WARNING: -wb is ignored with -abam
*****
*****WARNING: -wb is ignored with -abam*****
*****
**********
*****WARNING: -wb is ignored with -abam
*****WARNING: -wb is ignored with -abam*****WARNING: -wb is ignored with -abam
**********
*****
*****WARNING: -wb is ignored with -abam*****
**********WARNING: -wb is ignored with -abam
**********WARNING: -wb is ignored with -abam*****
*****
**********WARNING: -wb is ignored with -abam
*****
*****WARNING: -wb is ignored with -abam*****
*****
ready to performing differential AS analyses...
find: warning: you have specified the -maxdepth option after a non-option argument -type, but options are not positional (-maxdepth affects tests specified before it as well as those specified after it).  Please specify options before other arguments.

Best,
James

If it helps, the error message from shell script 3 is below:
(It might be an issue with the process_raw_AS_differential_output.pl script but I do not know enough perl to comment on any further)

bash /lab/app/JUM/JUM_170323/JUM_1.3.1/JUM_3.sh /lab/app/JUM/JUM_170323/JUM_1.3.1 adjusted_pvalue 0.05 6 3
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/process_raw_AS_differential_output.pl line 20.
awk: cmd. line:1: fatal: cannot open file `AS_differential.txt' for reading (No such file or directory)
AS_differential.txt: No such file or directory
AS_differential.txt: No such file or directory
AS_differential.txt: No such file or directory
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_adjusted_pvalue_0.05.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_total.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_adjusted_pvalue_0.05.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_total.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_adjusted_pvalue_0.05.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_total.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_adjusted_pvalue_0.05.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_total.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_adjusted_pvalue_0.05.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_total.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_adjusted_pvalue_0.05.txt) could not be opened. Exiting!
Error: The requested bed file (temp_long_intron_retention_junction_coordinate_total.txt) could not be opened. Exiting!
Can't locate Array/Utils.pm in @INC (@INC contains: /etc/perl /usr/local/lib/perl/5.14.2 /usr/local/share/perl/5.14.2 /usr/lib/perl5 /usr/share/perl5 /usr/lib/perl/5.14 /usr/share/perl/5.14 /usr/local/lib/site_perl .) at /lab/app/JUM/JUM_170323/JUM_1.3.1/screening_cassette_exon_events.pl line 4.
BEGIN failed--compilation aborted at /lab/app/JUM/JUM_170323/JUM_1.3.1/screening_cassette_exon_events.pl line 4.
Error: The requested bed file (cassette_exon_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S100_temp_total_cassette_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (cassette_exon_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S104_temp_total_cassette_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (cassette_exon_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S108_temp_total_cassette_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (cassette_exon_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S102_temp_total_cassette_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (cassette_exon_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `SC2_S106_temp_total_cassette_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (cassette_exon_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S110_temp_total_cassette_with_indicator.txt' for reading (No such file or directory)
awk: cmd. line:1: fatal: cannot open file `cassette_exon_coordinate.bed' for reading (No such file or directory)
Can't locate Array/Utils.pm in @INC (@INC contains: /etc/perl /usr/local/lib/perl/5.14.2 /usr/local/share/perl/5.14.2 /usr/lib/perl5 /usr/share/perl5 /usr/lib/perl/5.14 /usr/share/perl/5.14 /usr/local/lib/site_perl .) at /lab/app/JUM/JUM_170323/JUM_1.3.1/screening_MXE_events.pl line 4.
BEGIN failed--compilation aborted at /lab/app/JUM/JUM_170323/JUM_1.3.1/screening_MXE_events.pl line 4.
Error: The requested bed file (MXE_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S100_temp_total_MXE_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (MXE_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S104_temp_total_MXE_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (MXE_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S108_temp_total_MXE_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (MXE_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S102_temp_total_MXE_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (MXE_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S106_temp_total_MXE_with_indicator.txt' for reading (No such file or directory)
Error: The requested bed file (MXE_coordinate.bed) could not be opened. Exiting!
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/count_intron_read_long_intron_retention_step2.pl line 17.
can't open input1 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/determining_rightful_total_cassette_exon_event.pl line 17.
awk: cmd. line:1: fatal: cannot open file `S110_temp_total_MXE_with_indicator.txt' for reading (No such file or directory)
Can't locate Array/Utils.pm in @INC (@INC contains: /etc/perl /usr/local/lib/perl/5.14.2 /usr/local/share/perl/5.14.2 /usr/lib/perl5 /usr/share/perl5 /usr/lib/perl/5.14 /usr/share/perl/5.14 /usr/local/lib/site_perl .) at /lab/app/JUM/JUM_170323/JUM_1.3.1/screening_A5SS_A3SS_events.pl line 4.
BEGIN failed--compilation aborted at /lab/app/JUM/JUM_170323/JUM_1.3.1/screening_A5SS_A3SS_events.pl line 4.
awk: cmd. line:1: fatal: cannot open file `Valid_total_A5SS_event.txt' for reading (No such file or directory)
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/profiling_splicing_patterns_from_AS_events_4.pl line 18.
awk: cmd. line:1: fatal: cannot open file `Valid_total_A3SS_event.txt' for reading (No such file or directory)
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/profiling_splicing_patterns_from_AS_events_4.pl line 18.
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/profiling_splicing_patterns_from_AS_events_5.pl line 18.
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/profiling_splicing_patterns_from_AS_events_5.pl line 18.
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/profiling_splicing_patterns_from_AS_events_5.pl line 18.
can't open input2 file: No such file or directory at /lab/app/JUM/JUM_170323/JUM_1.3.1/profiling_splicing_patterns_from_AS_events_6.pl line 20.
mkdir: cannot create directory `FINAL_JUM_OUTPUT': File exists
mv: cannot stat `AS_differential_JUM_output_intron_retention_adjusted_pvalue_0.05.txt': No such file or directory
mv: cannot stat `*coordinate.bed': No such file or directory

**Comment by qqwang-arnold** [comment]
The warnings from running JUM_2.sh are all normal. No problem. 

The warnings from running JUM_3.sh: it seems that you did not run the R script after running JUM_2.sh. The R_script_JUM.R will generate a file called AS_differential.txt which is needed for JUM_3.sh to run. You can take a look at the JUM manual again for details

**Comment by jamespblloyd** [comment]
Thanks for getting back to me! 

Good to know that those warnings are normal for JUM_2.sh. 

I did run the R script:
Rscript /lab/app/JUM/JUM_170323/JUM_1.3.1/R_script_JUM.R experiment_design_pair1.txt > outputFile.Rout 2> errorFile.Rout 

But there must have been a problem with either than or something regarding the input earlier. I will get back to you when I have investigated the problem further.
